crypto/info.c

```diff
@@ -131,7 +131,7 @@ DEFINE_RUN_ONCE_STATIC(init_info_strings)
 #ifdef OPENSSL_RAND_SEED_NONE
         add_seeds_string("none");
 #endif
-#ifdef OPENSSL_RAND_SEED_RTDSC
+#ifdef OPENSSL_RAND_SEED_RDTSC
         add_seeds_string("stdsc");
```

Comment:
Actually the string also should be changed to `"rdtsc"`, this would be still within CLA: trivial.

Reply:
Thanks. Have made this change. (Of course, the rdtsc source still is prevented from actual use via various preventative mechanisms, so this PR merely fixes up the potential to display the compiled seed source.)